### PR TITLE
Paris

### PR DESCRIPTION
**Pull request by pariselectra**

### CODE REVIEW

File: client/src/pages/Form/Form.scss [hunk 1]
```diff
@@ -0,0 +1,46 @@
+.form{
+    background-color: #04566e;
```
Comment (richardjhong) [inline client/src/pages/Form/Form.scss:2]: please swap to var(--darkBackground)

File: client/src/pages/Form/Form.scss [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+    color: #eff2f6;
+    margin: auto;
+    margin-top: 10%;
+    width: 50%;
+    padding: 2%;
+    border: 4px solid #b4d340;
```
Comment (richardjhong) [inline client/src/pages/Form/Form.scss:8]: var(--lightBorder)

File: client/src/pages/Form/Form.scss [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+
+.submitButton {
+    display: block;
+    margin: 0 auto;
+    width: 10%;
+    margin-top: 2%;
+}
+
+.title {
+    color: #022831;
```
Comment (richardjhong) [inline client/src/pages/Form/Form.scss:39]: var(--dark)

File: client/src/pages/Form/backup.txt [hunk 4]
```diff
@@ -0,0 +1,94 @@
+import React from 'react';
+import './Form.scss';
+import airplane from '../../images/airplaneArt.png';
+
```
Comment (richardjhong) [inline client/src/pages/Form/backup.txt:4]: is this file being used any more? Form/index.js looks good now

**Review by MarkGATX — APPROVED** [review]
let's do this! *fingers crossed*

**Pull request merged**